charts/sourcegraph/templates/minio/minio.Secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if and .Values.minio.enabled (not .Values.minio.auth.existingSecret) }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: minio-auth
+  labels:
+    app: minio
+    deploy: sourcegraph
+    app.kubernetes.io/component: minio
+type: Opaque
+data:
+  user: {{ .Values.minio.auth.user | toString | b64enc | quote }}
```

Comment:
@michaellzc - What's your opinion on this Secret structure? Your proposal in https://github.com/sourcegraph/sourcegraph/issues/32694 was much more flexible. Would that be a better approach?

Reply:
I like your approach, it's more opinionated.

Nit-pick: What's your thought on renaming the key from `user` and `password` to `awsAccessKeyId`  and `awsSecretAccessKey`? Using the AWS S3's naming convention feels more natural to me